tests/e2e-evm/.solhint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "solhint:recommended"
+  "extends": "solhint:recommended",
+  "rules": {
+    "func-visibility": ["warn", { "ignoreConstructors": true }]
```

Comment:
[`true for Solidity >=0.7.0`](https://protofire.github.io/solhint/docs/rules/security/func-visibility.html)

Reply:
tyty I did not know about this configuration